RULEngine/Game/Game.py

```diff
@@ -96,23 +96,13 @@ def _update_players(self, vision_frame, delta):
         self._update_players_of_team(yellow_team, self.yellow_team, delta)
 
     def kalman_update_ball(self, vision_frame, delta):
-        kalman_list = []
-        for c in vision_frame:
-            kalman_list.append(c["ball"])
-        self.ball.kalman_update(kalman_list, delta)
+        self.ball.kalman_update(vision_frame["ball"], delta)
```

Comment:
It is the same. We could reduce the signature of the method by using it though. Will change.